app/src/main/java/com/napzak/market/presentation/explore/bottomSheet/SortBottomSheet.kt

```diff
@@ -0,0 +1,117 @@
+package com.napzak.market.presentation.explore.bottomSheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import com.napzak.market.core.designsystem.theme.NapzakMarketTheme
+import com.napzak.market.presentation.explore.type.SortType
+import com.napzak.market.R
+import com.napzak.market.R.string.check_button
+import com.napzak.market.core.common.extension.noRippleClickable
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SortBottomSheet(
+    selectedSortType: String,
+    onDismissRequest: () -> Unit,
+    onSortItemClick: (String) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(NapzakMarketTheme.colors.black70)
+            .noRippleClickable {
+                onDismissRequest()
+            },
```

Comment:
p2: 다음과 같이 사용할 수 있답니다
```suggestion
            .noRippleClickable (onDismissRequest),
```